.github/workflows/add_labeled_issues_to_project.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     - uses: aareet/move-labeled-or-milestoned-issue@v2.0
       with:
         action-token: "${{ secrets.vsphere_github_actions }}"
-        project-url: "https://github.com/orgs/terraform-providers/projects/12"
+        project-url: "https://github.com/terraform-providers/terraform-provider-vsphere/projects/8"
```

Comment:
Are these links going to be redirected to the `hashicorp` org if we move the repo there?

Reply:
Yes, Github handles the redirect indefinitely, but I'll update them once the move happens.